src/coffee/helpers/repeater.coffee

```diff
@@ -57,7 +57,9 @@ class Repeater
 
     debug '(re)-trying task, %d remaining attempts', remainingAttempts
     if remainingAttempts is 0
-      defer.reject new Error "Failed to retry the task after #{@_options.attempts} attempts. Cause: #{lastError.stack}"
+      defer.reject
```

Comment:
Message "after x amount of times" makes it possible to differentiate in logs if request has been repeated or not. Current output is not helpful as it swallows whole information like headers (correlation id), request, response, http status code, project key what make issue analysis impossible:

"Failed to retry the task after 10 attempts. Cause: HttpError: Bad Gateway\n at /app/node_modules/sphere-node-sdk/lib/services/base.js:511:22\n at InventoryEntryService.BaseService._wrapResponse (/app/node_modules/sphere-node-sdk/lib/services/base.js:516:11)\n at /app/node_modules/sphere-node-sdk/lib/services/base.js:375:40\n at /app/node_modules/sphere-node-sdk/lib/connect/rest.js:222:22\n at Request._callback (/app/node_modules/sphere-node-sdk/lib/connect/rest.js:164:14)\n at Request.self.callback (/app/node_modules/sphere-node-sdk/node_modules/request/request.js:360:22)\n at emitTwo (events.js:87:13)\n at Request.emit (events.js:172:7)\n at Request. (/app/node_modules/sphere-node-sdk/node_modules/request/request.js:1202:14)\n at emitOne (events.js:82:20)\n at Request.emit (events.js:169:7)\n at IncomingMessage. (/app/node_modules/sphere-node-sdk/node_modules/request/request.js:1150:12)\n at emitNone (events.js:72:20)\n at IncomingMessage.emit (events.js:166:7)\n at endReadableNT (_stream_readable.js:921:12)\n at nextTickCallbackWith2Args (node.js:442:9)\n at process._tickCallback (node.js:356:17)"

I rather see it as a fix than breaking change.